### DISCUSSION TITLE
Are Github Checks Down For Anybody Else

### DISCUSSION THREAD

**Discussion by jjhiggz** [community/community#35212] [Actions] [answered]
# [Bug] Checks not Working

Right now I'm running some pipeline testing through Github actions. It's been working great for a few months, but for some reason, without changing anything related to the github actions yml, the checks just stopped running. They aren't even showing up in my PR's as available options

![Screen Shot 2022-10-05 at 10 02 19 AM](https://user-images.githubusercontent.com/59105254/194107652-e3fb20df-4cd1-4804-b558-2a07502ffdea.png)

I tried to see if maybe I ran out of minutes, but I'm not even close (I'm 70 out of 2000) right now. To clarify it is not that my checks are running slow, they aren't even running at all.

1. I was just curious if this is happening to anybody else
2. If not, what process do I go through to solve this issue?

**Comment by jjhiggz** [comment, marked as answer]
According to https://www.githubstatus.com/ , It seems like it could be related to these updates
![Screen Shot 2022-10-05 at 10 10 33 AM](https://user-images.githubusercontent.com/59105254/194109335-a0c07c3e-90fd-4a23-9267-0020835a0c89.png)

**Reply by jjhiggz** [reply]
That was totally it! Working like a charm now

**Comment by rorymcstay** [comment]
experiencing the same here, have not committed to PR in ~1 month. Hopefully resolved with above

**Reply by rorymcstay** [reply]
Working for me shortly after this post

**Reply by jjhiggz** [reply]
@rorymcstay Working for me now too. That must have been it

**Comment by justisGipson** [comment]
same situation. Testing ci/cd pipelines. No checks are showing, no actions are running

**Reply by jjhiggz** [reply]
It should be working now, or at least it is for me

**Reply by justisGipson** [reply]
yup, actions appear to be back!
